client/components/Category.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Grid from '@material-ui/core/Grid';
+import Box from '@material-ui/core/Box';
+import Button from '@material-ui/core/Button';
+
+const subjects = [
+  { name: 'Math' },
+  { name: 'English' },
+  { name: 'Science' },
+  { name: 'Computer' },
+];
+
```

Comment:
We said we weren't going to allow users to create categories right? So all of the categories will be defined here? Is four what we're going with or should we have more?